src/orion/algo/pbt/pbt.py

```diff
@@ -0,0 +1,877 @@
+    completed trials at lowest fidelity. If a trial is broken at higher fidelity, the
+    original trial leading to the broken trial is examinated again for ``exploit`` and ``explore``.
+    If the broken trial was the result of a fork, then we backtrack to the trial that was dropped
+    during ``exploit`` in favor of the forked trial. If the broken trial was a promotion, then
+    we backtrack to the original trial that was promoted.
+
+    For more information on the algorithm,
+    see original paper at https://arxiv.org/abs/1711.09846.
+
+    Jaderberg, Max, et al. "Population based training of neural networks."
+    arXiv preprint, arXiv:1711.09846 (2017).
+
+    Notes
+    -----
+    It is important that the experiment using this algorithm has a working directory properly
+    set. The experiment's working dir serve as the base for the trial's working directories.
+
+    The trial's working directory is ``trial.working_dir``. Using ``trial.hash_params`` to
+    determine a unique working dir for the trial will result in working on a different directory
+    than the one copied by PBT, hence missing the copied model parameters.
+
+    Parameters
+    ----------
+    space: `orion.algo.space.Space`
+        Optimisation space with priors for each dimension.
+    seed: None, int or sequence of int
+        Seed for the random number generator used to sample new trials.
+        Default: ``None``
+    population_size: int, optional
+        Size of the population. No trial will be continued until there are `population_size`
+        trials executed until lowest fidelity. If a trial is broken during execution at lowest
+        fidelity, the algorithm will sample a new trial, keeping the population of *non-broken*
+        trials at `population_size`.  For efficiency it is better to have less workers running than
+        total population_size. Default: 50.
+    generations: int, optional
+        Number of generations, from lowest fidelity to highest one. This will determine how
+        many branchings occur during the execution of PBT. Default: 10
+    exploit: dict or None, optional
+        In the mutate part, one can define the customized mutate function with its mutate factors,
+        such as multiply factor (times/divides by a multiply factor) and add factor
+        (add/subtract by a multiply factor). The function must be defined by
+        an importable string. If None, default
+        mutate function is used: ``orion.algo.mutate_functions.default_mutate``.
+    explore: dict or None, optional
+        In the mutate part, one can define the customized mutate function with its mutate factors,
+        such as multiply factor (times/divides by a multiply factor) and add factor
+        (add/subtract by a multiply factor). The function must be defined by
+        an importable string. If None, default
+        mutate function is used: ``orion.algo.mutate_functions.default_mutate``.
```

Comment:
Same as `exploit`.

Reply:
Fixed in https://github.com/Epistimio/orion/pull/705/commits/6471414296e42671582dcdf125393f4e1c6a9f65